app/cells/plugins/core/tooltip_cell.rb

```diff
@@ -8,7 +8,7 @@ def show
       private
 
       def tooltip_id
-        @options[:id].parameterize('_')
+        @options[:id].parameterize({ separator: '_' })
```

Comment:
You shouldn't need to wrap the `separator` named parameter in a hash - you can just do `.parameterize(separator: '_')`